### PR TITLE
Readme updates

### LINKED ISSUE #58: Readme and Github pages
The first image in readme.md and the github pages isn't sourced properly.

### PR DESCRIPTION
**Pull request by AbhijithGanesh**
Fixed the issues with the images. All images from github assets can be seen on ReadMe.md .

fix #58

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
 There are many inconsistencies between browsers. Like Firefox 3 has a margin on top of paragraphs but Internet Explorer 7 doesn't have any margin. There are thousands of browsers with hundreds of versions. Each version at least has 500+ inconsistencies with different browsers' different versions. How to keep up? This is an easy to use solution called **reseter.css**
 
-![Browser Inconsistencies](https://github.com/krishdevdb/reseter.css/raw/master/.github/assets/css_reset.png)
+![Browser Inconsistencies](https://github.com/resetercss/reseter.css/blob/main/.github/assets/css_reset.png)
```
Comment (ghost) [inline readme.md:18]: This should be raw/master, Could you revert the blob/main

File: readme.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
...ithub"></a></p><img height="100px" width="100px" src="https://github.com/krishdevdb/reseter.css/raw/main/logo.png"><br><h1>Reseter.css</h1><h5>A Futuristic Alternative to Normalize.css and CSS Resets</h5><a href="https://twitter.com/intent/tweet?url=https://github.com/krishdevdb/reseter.css&text=Reseter.css%20helped%20me%20out%20in%20my%20website%20by%20normalizing%20everything.%20And%20Reseting%20The%20Browser%20Default%20Styles.%20Now%20My%20Website%20Looks%20The%20Same%20Across%20All%20Browsers"><img alt="Twitter URL" src="https://img.shields.io/twitter/url?color=%23ffffff00&logo=twitter&style=for-the-badge&url=https%3A%2F%2Fgithub.com%2Fkrishdevdb%2Freseter.css"></a><br><p><img src="https://github.com/resetercss/reseter.css/blob/main/.github/assets/showcase.png" height="400"></p><p><a href="https://krishdevdb.github.io/reseter.css">Website</a> · <a href="https://github.com/krishdevdb/reseter.css/issues/new?assignees=krishdevdb%27&labels=feature+request&template=feature-request.md&title=Request%3A+">Request a Feature</a> · <a href="https://github.com/krishdevdb/reseter.css/issues/new?assignees=krishdevdb&labels=bug&template=bug-report.md&title=Bug%3A+">Report a Bug</a></p></div>
```
Comment (ghost) [inline readme.md]: The badges aint fixed, you need to switch krishdevdb/reseter.css to resetercss/reseter.css I recommend using the search and replace feature in vscode/sublime/you idea

**Review by ghost — CHANGES_REQUESTED** [review]

File: readme.md (hunk 1 above)
Comment (AbhijithGanesh) [reply readme.md:18]: I haven't fully updated it is it? I shall patch the changes if you want

Comment (ghost) [reply readme.md:18]: You need to change this to 

```md
![Browser Inconsistencies](https://github.com/resetercss/reseter.css/raw/master/.github/assets/css_reset.png)
```

**Comment by AbhijithGanesh** [comment]
I have made the changes as requested. Can you please check and let me know?

**Review by AbhijithGanesh — COMMENTED** [review]
Please check the changes here, I don't know why it isn't reflecting there on the "Commits" Section @krishdevdb

File: readme.md [hunk 3]
```diff
@@ -67,7 +67,7 @@ There are many inconsistencies between browsers. Like Firefox 3 has a margin on
 
 * [CodeSandBox](https://codesandbox.io/s/resetercss-26kzw)
 
-* [CodePen](https://codepen.io/krishdevdb/pen/XWppdvK)
+* [CodePen](https://codepen.io/resetercss/pen/XWppdvK)
```
Comment (ghost) [inline readme.md]:
```suggestion
* [CodePen](https://codepen.io/krishdevdb/pen/XWppdvK)
```

**Review by ghost — CHANGES_REQUESTED** [review]

**Pull request merged**